magniv/utils/utils.py

```diff
@@ -34,12 +34,10 @@ def _get_function_from_json(key, filepath):
     :param filepath: the path to the json file that contains the function information
     :return: The location and name of the function
     """
-    f = open(filepath)
-    task_info_list = json.load(f)
-    for task in task_info_list:
-        if task["key"] == key:
-            break
-    return task["location"], task["name"]
+    with open(filepath) as fo:
```

Comment:
@seanabreau Honestly --- and this is my bad lol -- I don't think we use this function anymore, magniv-run now requires you to state explicitly where the function is to run it.

At one time you were able to do something like magniv-run get_emails and it will do a lookup in the dump.json and use this function to find where that get_emails task was and then run it.

Because the magniv-run task is being run on a kubernetes pod operator on some docker it didn't have the dump.json (bc the dump.json is being created on the build worker and doesn't need to be copied on the created docker images)

So yeah idt we use this function anymore --- it would be cool to bring it back to magniv-run <task key> but for now I think we can just leave it as is.

I am going to remove this function.

Reply:
gotcha thanks for that context makes sense